internal/pipe/release/body.go

```diff
@@ -67,5 +72,10 @@ func describeBody(ctx *context.Context) (bytes.Buffer, error) {
 		ReleaseNotes: ctx.ReleaseNotes,
 		DockerImages: dockers,
 	})
+	// Truncate the release notes if it's too long (github doesn't allow more than 125000 characters)
+	if out.Len() > maxReleaseNotesBodyLength {
+		out.Truncate(maxReleaseNotesBodyLength - len(ellipsis))
+		out.Write([]byte(ellipsis))
+	}
```

Comment:
since this is github-specific, maybe we should put it in the [`github` impl](https://github.com/goreleaser/goreleaser/blob/master/internal/client/github.go) only?

Reply:
yepp you are right, I changed the location to the github impl. Thanks for the input.